File: ordenação/Python/bubble_sort.py
vetor = [5, 3, 8, 4, 2]

tamanho = len(vetor)

# Percorre o vetor várias vezes
for i in range(tamanho - 1):

    # Compara elementos vizinhos
    for j in range(tamanho - 1 - i):

        # Se estiverem fora de ordem
        if vetor[j] > vetor[j + 1]:

            # Troca os elementos
            vetor[j], vetor[j + 1] = vetor[j + 1], vetor[j]

print("Vetor ordenado:", vetor)

File: ordenação/JavaScript/bubble_sort.js
let vetor = [5, 3, 8, 4, 2];

let tamanho = vetor.length;

// Percorre o vetor várias vezes
for (let i = 0; i < tamanho - 1; i++) {

    // Compara elementos vizinhos
    for (let j = 0; j < tamanho - 1 - i; j++) {

        // Se estiverem fora de ordem
        if (vetor[j] > vetor[j + 1]) {

            // Faz a troca
            let temp = vetor[j];
            vetor[j] = vetor[j + 1];
            vetor[j + 1] = temp;
        }
    }
}

console.log("Vetor ordenado:", vetor);
